File: lab-web-based-java-programming/practice-evaluation/src/main/java/com/app/pojos/Student.java
package com.app.pojos;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

@Entity
@Table(name = "students")
public class Student {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "roll_no")
	private Integer rollNo;
	@Column(length = 20, name = "first_name")
	private String firstName;
	@Column(length = 20, name = "last_name")
	private String lastName;
	@Column(length = 20, name = "city")
	private String city;
	@ManyToOne
	@JoinColumn(name = "course_name", nullable = false)
	private Course course;

	public Student() {
		super();
	}

	public Student(String firstName, String lastName, String city) {
		super();
		this.firstName = firstName;
		this.lastName = lastName;
		this.city = city;
	}

	public Integer getRollNo() {
		return rollNo;
	}

	public void setRollNo(Integer rollNo) {
		this.rollNo = rollNo;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public Course getCourse() {
		return course;
	}

	public void setCourse(Course course) {
		this.course = course;
	}

	@Override
	public String toString() {
		return "Student [rollNo=" + rollNo + ", firstName=" + firstName + ", lastName=" + lastName + ", city=" + city
				+ ", course=" + course + "]";
	}
}

File: practice-cpp-programming-language/practice-sunbeam-admission-process/core/Student.cpp
#include <iostream>

using namespace std;

class Student
{
private:
  string name;
  int rollNo;
  long mobile;

public:
  Student(string name, int rollNo, long mobile)
  {
    this->name = name;
    this->rollNo = rollNo;
    this->mobile = mobile;
  }

  string getName()
  {
    return this->name;
  }

  int getRollNo()
  {
    return this->rollNo;
  }

  long getMobile()
  {
    return this->mobile;
  }

  void setName(string name)
  {
    this->name = name;
  }

  void setRollNo(int rollNo)
  {
    this->rollNo = rollNo;
  }

  void setMobile(long mobile)
  {
    this->mobile = mobile;
  }

  void toString()
  {
    cout << "Student [" << this->getName() << ", " << this->getRollNo() << ", " << this->getMobile() << "]" << endl;
  }
};